tomviz/ModuleManager.cxx

```diff
@@ -345,6 +357,9 @@ bool ModuleManager::deserialize(const pugi::xml_node& ns, const QDir& stateDir)
   int numViews = 0, numLayouts = 0;
   for (pugi::xml_node node = ns.child("View"); node; node = node.next_sibling("View"))
   {
+    pugi::xml_node agnode = node.child("AxesGrid");
```

Comment:
These variables are not very symmetric with those above, and the capitalization is off. I would suggest using axesGridNode as above so the relation is clear, and the meaning easier to see at a glance.